test/harvest/test_distill.py

```diff
@@ -378,6 +378,48 @@ def test_apc_openalex_fallback(test_session, snapshot):
     assert _pub(session).apc == 1234
 
 
+def test_apc_negative(test_session, snapshot):
+    """
+    negative apc values are not returned
+    """
+    with test_session.begin() as session:
+        session.bulk_save_objects(
+            [
+                Publication(
+                    doi="10.1515/9781503624153",
+                    openalex_json={
+                        "apc_paid": {"value_usd": -123},
+                    },
+                ),
+            ]
+        )
+
+    distill(snapshot)
+
+    assert _pub(session).apc is None
+
+
+def test_apc_not_a_number(test_session, snapshot):
+    """
+    negative apc values are not returned
```

Comment:
Update to docstring that describes this test?

Reply:
good catch, fixed